### PR TITLE
Add covid content changes (NI)

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?

Adds new interstitial pages and updated content related to the coronavirus for the Northern Ireland household and individual schemas.

### How to review

Check the changes match the spreadsheet attached to the [trello card.](https://trello.com/c/ErQUbzWQ)

### Checklist

- [x] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-covid-changes/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-covid-changes/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by petechd** [comment]
[Covid changes NI HH/HI (S)](https://trello.com/c/ErQUbzWQ/4220-covid-changes-ni-hh-hi-s)

File: source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet [hunk 1]
```diff
@@ -1,8 +1,38 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local question(title) = {
-  title: title,
+local questionTitle(isProxy, isEmployed) = (
```
Comment (LJBabbage) [inline source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet:4]: I find this change hard to read, particularly the else, else. Any reason I am missing why it has been changed? I was looking for 'if the' to be added

**Review by LJBabbage — CHANGES_REQUESTED** [review]
Bar the one comment this covers all the changes (I did check content) and works as expected. There is a couple of mistakes in the spreadsheet around remove and edit, but they are correct on this PR

File: source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+  )
+);
+
+local questionDescription(isProxy, isEmployed) = (
+  if isEmployed then (
+    if isProxy then
+      ['If the <strong>coronavirus</strong> pandemic has affected their working hours, select the answer that best describes their <strong>current circumstances</strong>.']
+    else ['If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>.']
+  ) else []
+);
```
Comment (LJBabbage) [inline source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet:32]: I find this change hard to read, particularly the else, else. Any reason I am missing why it has been changed? I was looking for 'if the' to be added

Comment (petechd) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet:32]: This is how we do such variants in jsonnet style guide, so it's up to date with the latest recommendations. I expected this change to be requested in review so I refactored in a same way @berroar did in his covid ENG/WLS content changes PR.

Comment (LJBabbage) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet:32]: Can we at least change the formatting, so the else's don't line up and it's a bit easier on the eyes

Comment (MebinAbraham) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet:32]: Jsonnet format linter isn't very strict. I wonder if the style guide should be to use multiple variables to break it down to something like this which is a bit more digestible:
```jsonnet
local questionTitle(isProxy, isEmployed) = (
  local employedTitle = (
    if isProxy then
      {
        text: 'In <em>{person_name_possessive}</em> main job, how many hours a week do they usually work?',
        placeholders: [
          placeholders.personNamePossessive,
        ],
      }
    else 'In your main job, how many hours a week do you usually work?'
  );

  local unemployedTitle = (
    if isProxy then
      {
        text: 'In <em>{person_name_possessive}</em> main job, how many hours a week did they usually work?',
        placeholders: [
          placeholders.personNamePossessive,
        ],
      }
    else 'In your main job, how many hours a week did you usually work?'
  );

  if isEmployed then employedTitle else unemployedTitle
);
```
Plus isEmployed/isProxy could be flipped on our preference.

Comment (LJBabbage) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet:32]: My eyes thank you Mebin

Comment (petechd) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet]: Ok, refactored, thanks. Do we need to update jsonnet style guide now?

Comment (MebinAbraham) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet]: We should but prob suits a new PR with that change where all devs can review it?

Comment (petechd) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet]: Can always reset hard and force push😉

File: source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet [hunk 3]
```diff
@@ -1,8 +1,41 @@
...
+  if isEmployed then employedTitle else unemployedTitle
+);
+
+local questionDescription(isProxy, isEmployed) = (
+  if isEmployed then (
+    if isProxy then
+      ['If the <strong>coronavirus</strong> pandemic has affected their working hours, select the answer that best describes their <strong>current circumstances</strong>.']
+    else ['If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>.']
+  ) else []
+);
```
Comment (MebinAbraham) [inline source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet:35]: This should be made consistent with questionTitle.
```jsonnet
local questionDescription(isProxy, isEmployed) = (
  local employedDescription = (
    if isProxy then
      ['If the <strong>coronavirus</strong> pandemic has affected their working hours, select the answer that best describes their <strong>current circumstances</strong>.']
    else
      ['If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>.']
  );

  if isEmployed then employedDescription else []
);
```

Comment (petechd) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet]: Done, thanks!

**Review by MebinAbraham — APPROVED** [review]

**Comment by MebinAbraham** [comment]
We should raise a card to update the style guide or just make a PR.

**Review by LJBabbage — APPROVED** [review]

**Pull request merged**